### DISCUSSION TITLE
Already approved - Expediting academic benefits

### DISCUSSION THREAD

**Discussion by obiwan138** [community/community#139015] [GitHub Education]
### Hi everyone,

Thank you for taking time to read this !

I uploaded proof that I am a student which was accepted, and I am wondering if anything is possible to get access to student benefits ASAP, as it would really help in one my classes.
I understood from FAQ that this is maybe possible

**Thank you !**

Thomas

**Comment by davevad93** [comment]
Hi @ThomasEtheve ,

The [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) state the following: 
"***If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you.***"

That means someone from the Education team could take a look at your discussion and expedite the process for you, however, given the overwhelming amount of daily requests for the `Student Developer Pack`, it's not always possible to do so.

Generally, it takes a few days for the benefits to become available, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ThomasEtheve  was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721
